scylla/src/lib.rs

```diff
@@ -131,8 +131,84 @@ pub mod frame {
 }
 
 /// Serializing bound values of a query to be sent to the DB.
+// Note: When editing comment on submodules here edit corresponding comments
+// on scylla-cql modules too.
 pub mod serialize {
-    pub use scylla_cql::types::serialize::*;
+    pub use scylla_cql::types::serialize::SerializationError;
+    /// Contains the [BatchValues][batch::BatchValues] and [BatchValuesIterator][batch::BatchValuesIterator] trait and their
+    /// implementations.
```

Comment:
Oh, so `[][]` works as well? I only knew about `[]()`.

Reply:
> Actually you will notice that my third commit in this PR removes an occurence of `[]()` because it was rendered incorrectly.

It was not working because `[]` and `()` were on a separate line (separated by `//!`):
```rust
//! But the driver will accept anything implementing the trait [SerializeRow]
//! (crate::serialize::row::SerializeRow)
```

I even checked out locally to see, because at first glance, this looked correct to me.

Reply:
That makes sense. The version after my fix works too, right? At least it did when I checked locally.

Reply:
Yes, for me it renders correctly as well

Reply:
@muzarski do you know why 
```
[batch::BatchValues]
[`batch::BatchValues`]
```
don't work? This of course applies to all the other submodules in this place.

Reply:
I'm not really sure. But actually, I see that `[BatchValues][batch::BatchValues]` does not work as well...

![image](https://github.com/user-attachments/assets/ef9a5cd0-b30b-4de3-a6c7-e620c7649c08)

Reply:
The colors in vscode are not working great. Something can be in brighter green and still not render correctly (or being in darker green and render correctly).
Check using cargo doc.

Reply:
Oh, wasn't aware of that.

Reply:
So it looks like things that are highlighted in vscode, do not render correctly for `cargo doc` and vice-versa...
![image](https://github.com/user-attachments/assets/0dbd302f-5cd7-42b9-a368-50d2aa73babf)

Reply:
`[]()` should work everywhere, so let's just stick to it. It's the ordinary Markdown format.

Reply:
> `[]()` should work everywhere, so let's just stick to it. It's the ordinary Markdown format.

I'll do it. Still it's sad that this is so difficult to get right.

Reply:
Hmm.. I can change it but both `[BatchValues](batch::BatchValues)` and `[BatchValues][batch::BatchValues]` are not highlighted in VSCode (and both render corrrectly), so there is no difference.